cpp/src/gandiva/precompiled/decimal_ops_test.cc

```diff
@@ -887,9 +893,11 @@ TEST_F(TestDecimalSql, FromDouble) {
       std::make_tuple(DecimalScalar128{1230, 38, 33}, 1.23E-30, false),
       std::make_tuple(DecimalScalar128{123, 38, 38}, 1.23E-36, false),
 
-      // overflow due to very low double
-      std::make_tuple(DecimalScalar128{0, 0, 38, 0}, std::numeric_limits<double>::min(),
-                      true),
+      // TODO: std::round() converts this to 0.
```

Comment:
Why is it a problem?

Reply:
you are right. I incorrectly thought that that double_min is a large negative number (like int_min). fixed the tests now. thanks !